settings.gradle

```diff
@@ -29,3 +25,21 @@ include 'test-suite-kotlin'
 include 'grpc-health'
 include 'grpc-opentracing'
 
+enableFeaturePreview 'TYPESAFE_PROJECT_ACCESSORS'
+
+dependencyResolutionManagement {
+    repositories {
+        mavenCentral()
+        maven {
+            url "https://s01.oss.sonatype.org/content/repositories/snapshots/"
+            mavenContent {
+                snapshotsOnly()
+            }
+        }
+    }
+    versionCatalogs {
+        mn {
+            from "io.micronaut:micronaut-bom:${providers.gradleProperty('micronautVersion').get()}"
+        }
+    }
```

Comment:
This is no longer idiomatic for Micronaut module builds. Remove `dependencyResolutionManagement.versionCatalogs` and use the build plugin instead. See https://github.com/micronaut-projects/micronaut-project-template/blob/09c49b0c2fcc7692c6eda6b79f24775a41ff1595/settings.gradle#L17

```
micronautBuild {
    importMicronautCatalog()
}
```